cdc/processor/pipeline/actor_node.go

```diff
@@ -54,9 +55,15 @@ func (n *ActorNode) TryRun(ctx context.Context) error {
 			return errors.Trace(err)
 		}
 
-		if ok {
-			n.messageStash = nil
-		} else {
+		// node is blocked
+		if !ok {
+			return nil
+		}
+
+		n.messageStash = nil
+		processedCount++
+		// processed too many messages, not block other tables
+		if processedCount >= defaultOutputChannelSize {
```

Comment:
Could you elaborate the scenario you want to address?

Reply:
done